test/e2e/util.ts

```diff
@@ -111,8 +111,11 @@ export class E2eHelper {
   // Return id of test workspaces(e2e-tests. Please look devfile-example.yaml file)
   async getWorkspaceId(): Promise<any> {
     const workspaces = await this.getAllWorkspaces()
-    const workspaceId = workspaces.filter((wks => wks.name.match(this.devfileName))).map(({ id }) => id)[0]
+    if (workspaces.length === 0) {
+      throw Error('Workspace not found')
```

Comment:
I am not sure, but since we introducing `newError` method we should use it. Otherwise it might be a mess of `new Error(...)` and `new Error()` calls.

Reply:
A new function is introduced to preserver an original stack trace and message.
So, not for this case